onnxscript/ir/_convenience/_constructors_test.py

```diff
@@ -1,20 +1,20 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
-"""Unit tests for the _convenience module."""
+"""Unit tests for the _constructors module."""
 
 import unittest
 
 import numpy as np
 
-from onnxscript.ir import _convenience
+from onnxscript.ir._convenience import _constructors
 
 
 class ConvenienceTest(unittest.TestCase):
     def test_tensor_accepts_torch_tensor(self):
         import torch as some_random_name  # pylint: disable=import-outside-toplevel
 
         torch_tensor = some_random_name.tensor([1, 2, 3])
-        tensor = _convenience.tensor(torch_tensor)
+        tensor = _constructors.tensor(torch_tensor)
```

Comment:
nit: Maybe a test showing usage of _constructors,node

Reply:
Added doctests